tests/e2e/csi_snapshot_basic.go

```diff
@@ -3879,36 +3755,28 @@ var _ = ginkgo.Describe("Volume Snapshot Basic Test", func() {
 		defer func() {
 			ginkgo.By("Rebooting VC")
 			err = invokeVCenterReboot(ctx, vcAddress)
-			isVcRebooted = true
 			gomega.Expect(err).NotTo(gomega.HaveOccurred())
 			err = waitForHostToBeUp(e2eVSphere.Config.Global.VCenterHostname)
 			gomega.Expect(err).NotTo(gomega.HaveOccurred())
 			ginkgo.By("Done with reboot")
-			essentialServices := []string{spsServiceName, vsanhealthServiceName, vpxdServiceName}
-			checkVcenterServicesRunning(ctx, vcAddress, essentialServices)
 
 			// After reboot.
 			bootstrap()
-
-			framework.Logf("Deleting volume snapshot")
-			for _, snapshot := range volumesnapshots {
-				deleteVolumeSnapshotWithPandoraWait(ctx, snapc, namespace, snapshot.Name, pandoraSyncWaitTime)
-			}
 		}()
 
 		ginkgo.By("Rebooting VC")
 		err = invokeVCenterReboot(ctx, vcAddress)
-		isVcRebooted = true
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
 		err = waitForHostToBeUp(e2eVSphere.Config.Global.VCenterHostname)
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
 		ginkgo.By("Done with reboot")
-		essentialServices := []string{spsServiceName, vsanhealthServiceName, vpxdServiceName}
-		checkVcenterServicesRunning(ctx, vcAddress, essentialServices)
 
 		// After reboot.
 		bootstrap()
 
+		ginkgo.By("Waiting for 15 mins post vs reboot for testbed to reach healthy state")
```

Comment:
Don't put static wait time. edit the stopped services required for 9.0

Reply:
done